### ISSUE TITLE
Stalls Firefox after interaction with the extension UI.

### ISSUE THREAD

**Issue by JacobTravers** [vantezzen/skip-silence#124]
**Describe the bug**
Seems to bog down firefox as a whole after interacting with the extension's UI. I had to kill the browser process and restart firefox. I have reproduced this after re-installing the extension multiple times. The same behavior also occurs on my laptop, the OS, Browser, and Versions are identical on that machine.

**To Reproduce**
1. Open Firefox
2. Go to Youtube
3. Click on the Skip Silence extension icon in the toolbar
4. Adjust any setting
5. Return to any part of the Firefox Browser
6. See that it is unresponsive, or hangs / delays.

**Expected behavior**
The previous behavior of the extension, where it worked normally and did not bog down firefox.

**Desktop (please complete the following information):**
 - OS: Arch Linux 5.18.16-zen1-1-zen
 - Browser Mozilla Firefox 104.0b9 (Developer Edition)
 - Skip Silence Version 5.0.7

**Comment by vantezzen** [comment]
I hoped the last update would have fixed this but it seems to still be an issue... the content script and popup script constantly seem to bounce config updates back and forth to create an infinite loop.

I'll look into this, for the time being you can downgrade to 4.0.0 at https://github.com/vantezzen/skip-silence/releases/tag/4.0.0.

**Comment by JacobTravers** [comment]
@vantezzen for the time being can you roll back the version listed on firefox addons?
I tried installing the zip from that release but it doesn't seem to work.

**Comment by vantezzen** [comment]
Unfortunately not - Mozilla pulled every previous version from the store, that's why it wasn't in Mozilla Add-ons for 4 months. They reviewed the code as fine but they couldn't build the extension because they didn't follow the build instructions. The whole history is available at https://github.com/vantezzen/skip-silence/issues/112 😅

I created video instructions over at  https://github.com/vantezzen/skip-silence/issues/112#issuecomment-1216448439 if that can help.

I refactored some parts of the synchronization code but couldn't find the reason it fails on Firefox yet
